src/driver/mod.rs

```diff
@@ -121,6 +121,21 @@ pub trait Poller {
         let entry = unsafe { entry.assume_init() };
         Ok(entry)
     }
+
+    /// Poll the driver and get multiple entries back.
+    ///
+    /// See [`Poller::poll`].
+    fn poll_entries<const N: usize>(
+        &mut self,
+        timeout: Option<Duration>,
+    ) -> io::Result<Entries<N>> {
+        let mut entries = Entries::new();
+        let polled = self.poll(timeout, entries.as_mut_slice())?;
+        unsafe {
+            entries.set_len(polled);
+        }
+        Ok(entries)
+    }
```

Comment:
On mio/io-ring the interface doesn't allow to process all completed entries without doing another poll.

Reply:
Yes. That's a compromise in design. The cqueue does not mean to use in most cases, but just to prevent errors. In most cases, we can avoid heap allocations.

Reply:
I think it's ok as long as a driver user can bound outstanding io operations by fixed size.